backend/app/dependencies.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import Depends, HTTPException, status
```

Comment:
I think /core/helpers might be a better location for this file. And I would also name it something like admin functionality. Would not group all the dependencies together.

Reply:
Usually, all generic dependencies are kept in a file called `dependencies.py`. It allows you to figure out which dependencies exist in your app and manage them more easily, especially when one dependency depends on another.

Reply:
Sure, but all our auth dependencies are in core/auth. So these are not in dependencies.py
Maybe we should have a access, permissions or rights folder? Where we put the dependencies on access level in?
I'm not against a global dependencies.py file, just not sure if this should be in it

Reply:
It looks more readable to me when I see `dependencies` files in the app root that includes all the global dependencies.

https://fastapi.tiangolo.com/tutorial/bigger-applications/#an-example-file-structure